### PR TITLE
chore: hide enable flags from schema and config examples

### PR DESCRIPTION
**Pull request by thalesmg**
Fixes https://emqx.atlassian.net/browse/EMQX-12730

Release version: v/e5.8

```sh
ͳ jq 'path(.. | select(.text? == "enable")) as $path | getpath($path | . += ["hash"])' < _build/docgen/emqx-enterprise/schema-v2-en.json
```

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

**Comment by zmstone** [comment]
Some of the enable flags having default value `false` will be problematic if it's not documented and not provided in the examples either. 
To avoid this, we should unify all the enable flags to have default value `true`, for the ones with `default => false` today, we should change the parent fields' schema to union with `disabled` and set its default value to `disabled`.

File: rel/config/examples/gateway.exproto.conf.example [hunk 1]
```diff
@@ -16,7 +16,6 @@ gateway.exproto {
     ## Configurations for request to ConnectionHandler service
     handler {
         address = "http://127.0.0.1:9001"
-        ssl_options {enable = false}
```
Comment (zmstone) [inline rel/config/examples/gateway.exproto.conf.example:19]: we'll need to keep this for now.

File: rel/config/examples/plugins.conf.example [hunk 2]
```diff
@@ -9,10 +9,8 @@ plugins {
             ##     Format: {name}-{version}
             ##     Note: name and version should be what it is in the plugin application
             name_vsn = "my_acl-0.1.0",
-
-            enable = true ## enable this plugin
         },
-        {name_vsn = "my_rule-0.1.1", enable = false}
```
Comment (zmstone) [inline rel/config/examples/plugins.conf.example:15]: we'll need to keep this as an example to show how to disable a plugin.

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-3446)

**Pull request merged**